features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/BenchmarkExt.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.internal.recorder
+
+import com.datadog.android.internal.profiler.BenchmarkSpan
+import com.datadog.android.internal.profiler.withinBenchmarkSpan
+
+private const val ATTRIBUTE_CONTAINER = "attribute.container"
+
+/**
+ * A wrap function of [withinBenchmarkSpan] dedicated to session replay span recording.
+ */
+internal inline fun <T : Any?> measureSpan(
```

Comment:
shouldn't function name be aligned with the name of the function it wraps? it essentially just adds one single attribute

Reply:
updated